### DISCUSSION TITLE
Problem with the Alamo installation

### DISCUSSION THREAD

**Discussion by SoumyaBandyopadhyay** [AMReX-Codes/amrex#2710] [Q&A] [answered]
Hello,
I tried to install ALAMO. I followed the similar steps mentioned in the 'Quick Start' . It succesfully configured, but after running 'Make'
for a while it ended up showing the following error:
![error1](https://user-images.githubusercontent.com/12543293/163107858-33d20728-fabe-485f-865a-e537a51650e4.png)
I am quite sure all the required dependencies are installed properly and the required folders, files are also present.
Still not getting the reason for this error. It will be very helpful if I get some suggestions regarding this issue.

**Comment by bsrunnels** [comment, marked as answer]
Hello, this question is related to Alamo specifically, not AMReX.  Please post your question on the Alamo github page https://github.com/solidsgroup/alamo
